addons/overpressure/functions/fnc_overpressureDamage.sqf

```diff
@@ -56,12 +56,13 @@ TRACE_3("cache",_overpressureAngle,_overpressureRange,_overpressureDamage);
             // If the target is the ACE_player
             if (_x == ACE_player) then {[_damage * 100] call BIS_fnc_bloodEffect};
 
-            if (isClass (configFile >> "CfgPatches" >> "ACE_Medical")) then {
-                [_x, _damage, "body", "backblast", _firer] call EFUNC(medical,addDamageToUnit);
-            } else {
-                TRACE_1("",isDamageAllowed _x);
-                if (!isDamageAllowed _x) exitWith {}; // Skip damage if not allowed
-                _x setDamage (damage _x + _damage);
+            TRACE_1("",isDamageAllowed _x);
+            if (isDamageAllowed _x) then {
```

Comment:
I would also try to catch the ACE Medial variable
```suggestion
            if (isDamageAllowed _unit && {_unit getVariable [QEGVAR(medical,allowDamage), true]}) then {
```

Reply:
That variable is never read by the game anywhere else as far as I can tell.

It is only set here:
https://github.com/acemod/ACE3/blob/master/addons/spectator/functions/fnc_setSpectator.sqf#L107
but that also should be deleted, as it does nothing.

What even is the point of that variable?

Reply:
_unit -> _x

Reply:
ups copy-pasted it from somewhere else :P